blocks/byline-block/features/byline/default.jsx

```diff
@@ -1,125 +1,43 @@
-import React, { Component } from 'react';
-import Consumer from 'fusion:consumer';
+import React from 'react';
 import PropTypes from 'prop-types';
-import styled from 'styled-components';
-import getThemeStyle from 'fusion:themes';
-import getProperties from 'fusion:properties';
-import getTranslatedPhrases from 'fusion:intl';
-import './byline.scss';
 
-const BylineSection = styled.section`
-  font-family: ${(props) => props.secondaryFont};
-  font-size: 1.25rem;
-  line-height: 2.5rem;
-  color: #191919;
-  ${({ stylesFor }) => stylesFor === 'list' && `
-    display: inline;
-    font-size: .875rem;
-    line-height: 1rem;
-  `}
-`;
-
-const By = styled.span`
-  ${({ stylesFor }) => stylesFor === 'list' && `
-    color: #3B3B3B;
-    margin-right: 0;
-  `}
-`;
-
-const BylineNames = styled.span`
-  ${({ stylesFor }) => stylesFor === 'list' && `
-    color: #434343;
-  `}
-`;
-
-@Consumer
-class ArticleByline extends Component {
-  constructor(props) {
-    super(props);
-    const {
-      globalContent, story, arcSite,
-    } = props;
+import { useFusionContext } from 'fusion:context';
+import { Byline } from '@wpmedia/shared-styles';
+
+const ArticleByline = ({ story, stylesFor = '', separator = false }) => {
+  const { globalContent } = useFusionContext();
```

Comment:
This should not be needed as the shared styles `<Byline` component will look for globalContent